### PR TITLE
pickle

### PR DESCRIPTION
**Pull request by DrFineSir**

### CODE REVIEW

File: pickle [hunk 1]
```diff
@@ -0,0 +1 @@
+8=============)~~~~
```
Comment (Synqat) [inline pickle:1]: this looks bad i think

**Review by Synqat — CHANGES_REQUESTED** [review]

File: pickle (hunk 1 above)
Comment (Lizrard) [reply pickle:1]: I do concur

Comment (DrFineSir) [reply pickle:1]: this is litty i know

**Comment by DrFineSir** [comment]
MERGE OR FACE CONSEQUENCES

**Review by Synqat — APPROVED** [review]
thx

**Review by Lizrard — APPROVED** [review]
great pickle

**Pull request merged**